src/java/org/apache/cassandra/io/sstable/format/SSTableWriter.java

```diff
@@ -85,6 +85,7 @@ public abstract class SSTableWriter extends SSTable implements Transactional
     protected final MetadataCollector metadataCollector;
     protected final SerializationHeader header;
     protected final Collection<SSTableFlushObserver> observers;
+    private static boolean isInternalKeyspace;
```

Comment:
Why static?

Reply:
The only reason I had to make it static was because the method below where isInternalKeyspace is being used has a static context 
<img width="913" alt="Screenshot 2022-08-09 at 17 56 10" src="https://user-images.githubusercontent.com/102972224/183711731-3bbdb72f-0451-4533-bad2-196c1de6c99e.png">

Reply:
Yes, you can and should make other changes. This cannot be static then.